sound/effect.h

```diff
@@ -712,14 +712,14 @@ EFFECT(preon);
 EFFECT(pstoff);
 
 // Monophonic fonts
+
+// Idle effect, plays when blade is off.
 #ifdef ENABLE_IDLE_SOUND
 EFFECT2(idle, idle);
-EFFECT2(boot, idle);
-EFFECT2(font, idle);
-#else
-EFFECT(boot);
-EFFECT(font);      // also polyphonic
+EFFECT2(bgnidle, idle);
 #endif
+EFFECT(boot);
```

Comment:
Why can't we have these linked to bgnidle?

Reply:
Well before I know that a missing bgn sound would auto-skip to the base sound, this was needed. Reverted now because it works fine if no bgnidle sound exists.
It's great.